src/main/java/uk/gov/justice/hmpps/prison/repository/sql/InmateRepositorySql.kt

```diff
@@ -3,30 +3,31 @@ package uk.gov.justice.hmpps.prison.repository.sql
 enum class InmateRepositorySql(val sql: String) {
   FIND_INMATE_DETAIL(
     """
-        SELECT B.OFFENDER_BOOK_ID,
+        SELECT B.OFFENDER_BOOK_ID BOOKING_ID,
         B.BOOKING_NO,
         O.OFFENDER_ID,
         O.ROOT_OFFENDER_ID,
-        O.OFFENDER_ID_DISPLAY,
+        O.OFFENDER_ID_DISPLAY OFFENDER_NO,
         O.FIRST_NAME,
         CONCAT(O.middle_name, CASE WHEN middle_name_2 IS NOT NULL THEN concat(' ', O.middle_name_2) ELSE '' END) MIDDLE_NAME,
         O.LAST_NAME,
-        B.AGY_LOC_ID,
-        B.LIVING_UNIT_ID,
-        B.ACTIVE_FLAG,
+        B.AGY_LOC_ID AGENCY_ID,
+        B.LIVING_UNIT_ID ASSIGNED_LIVING_UNIT_ID,
+        CASE WHEN B.ACTIVE_FLAG = 'Y' THEN 1 ELSE 0 END ACTIVE_FLAG,
         B.IN_OUT_STATUS,
         B.STATUS_REASON,
         (SELECT MAX(OI.OFFENDER_IMAGE_ID)
-        FROM OFFENDER_IMAGES OI
-        WHERE OI.ACTIVE_FLAG = 'Y'
+                FROM OFFENDER_IMAGES OI
+                WHERE OI.ACTIVE_FLAG = 'Y'
         AND IMAGE_OBJECT_TYPE = 'OFF_BKG'
         AND OI.OFFENDER_BOOK_ID = B.OFFENDER_BOOK_ID
-        AND OI.IMAGE_VIEW_TYPE = 'FACE'
-        AND OI.ORIENTATION_TYPE = 'FRONT') AS FACE_IMAGE_ID,
-        O.BIRTH_DATE,
+                AND OI.IMAGE_VIEW_TYPE = 'FACE'
+        AND OI.ORIENTATION_TYPE = 'FRONT') AS FACIAL_IMAGE_ID,
+        O.BIRTH_DATE DATE_OF_BIRTH,
         O.BIRTH_PLACE,
         O.BIRTH_COUNTRY_CODE,
-        B.ASSIGNED_STAFF_ID AS ASSIGNED_OFFICER_ID
+        B.ASSIGNED_STAFF_ID AS ASSIGNED_OFFICER_ID,
+        B.BOOKING_BEGIN_DATE AS RECEPTION_DATE
```

Comment:
I am confused, so was reception date always missing? I though it was present sometimes?

Reply:
Yes present in other endpoint